samples/AnomalyDetector/multivariate/models.cadl

```diff
@@ -51,6 +51,7 @@ enum ModelStatus {
 model MultivariateDetectionResult {
   @doc("Result identifier, which is used to fetch the results of an inference call.")
   @key
+  @visibility("read")
   @format("uuid")
```

Comment:
How do we update this file? Manually?

Reply:
Yes. it is our sample project. But we still need ask service team to update their cadl in azure-rest-api-specs before we update  the new emitter version for azure-sdk-for-net.

Reply:
And what is the purpose of adding this line? I saw you add this at other files.

Reply:
0.39 cadl library add a lint rule for this. when you add @key decorator for a property must set @visibility.